test/optimisation/Optimisation.jl

```diff
@@ -0,0 +1,557 @@
+@testset "optimisation/Optimisation.jl" begin
```

Comment:
```suggestion
@testset "Optimisation" begin
```

Reply:
You can argue that the pattern you're suggesting is a better one:) But AFAIK we're not following this pattern elsewhere in the code, so I think consistency is a bit more important atm.

Reply:
Done.